notifications_utils/letter_timings.py

```diff
@@ -27,7 +28,55 @@ def set_gmt_hour(day, hour):
     return day.astimezone(pytz.timezone('Europe/London')).replace(hour=hour, minute=0).astimezone(pytz.utc)
 
 
-def get_letter_timings(upload_time, postage='second'):
+def get_next_work_day(date, non_working_days):
+    next_day = date + timedelta(days=1)
+    if non_working_days.is_work_day(
+        date=next_day.date(),
+        division=BankHolidays.ENGLAND_AND_WALES,
+    ):
+        return next_day
+    return get_next_work_day(next_day, non_working_days)
+
+
+def get_next_dvla_working_day(date):
+    """
+    Printing takes place monday to friday, excluding bank holidays
+    """
+    return get_next_work_day(date, non_working_days=non_working_days_dvla)
+
+
+def get_next_royal_mail_working_day(date):
+    """
+    Royal mail deliver letters on monday to saturday
+    """
+    return get_next_work_day(date, non_working_days=non_working_days_royal_mail)
+
+
+def get_delivery_day(date, *, days_to_deliver):
+    next_day = get_next_royal_mail_working_day(date)
+    if days_to_deliver == 1:
+        return next_day
+    return get_delivery_day(next_day, days_to_deliver=(days_to_deliver - 1))
+
+
+def get_min_and_max_days_in_transit(postage):
+    return {
+        # first class post is printed earlier in the day, so will
+        # actually transit on the printing day, and be posted the next
```

Comment:
Optional: Small nitpick, is it actually that first class is printed earlier in the day so will actually transit on the printing day and therefore be *delivered* the next day? The word posted to me implies the transit period

Reply:
Yeah, I think the original comment I copied maybe meant ‘posted through the letterbox’ but delivered is clearer
```suggestion
        # actually transit on the printing day, and be delivered the next
```